### DISCUSSION TITLE
base.go.kubebuilder.io/v4

### DISCUSSION THREAD

**Discussion by lexue100** [kubernetes-sigs/kubebuilder#3872] [Support]
go : 1.21.9 darwin/amd64
kubebuilder: 3.10.0
执行：kubebuilder  create api  时，一直报：failed to create API: unable to run post-scaffold tasks of "base.go.kubebuilder.io/v4": exit status 2

**Comment by camilamacedo86** [comment]
Hi, 

This go version is not support by the Kubebuilder version used.
See: https://github.com/kubernetes-sigs/kubebuilder/blob/v3.10.0/go.mod#L3

Also, see: https://kubebuilder.io/quick-start#versions-and-supportability

<img width="823" alt="Screenshot 2024-04-24 at 07 02 46" src="https://github.com/kubernetes-sigs/kubebuilder/assets/7708031/cfeaa8f0-85aa-42d1-b21e-e887e3d6b99a">

So, I would suggest you download the latest release of the Kubebuilder CLI or downgrade the go version.

Closing this one as sorted out.
